cluster/manifests/03-ebs-csi/ebs-controller.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ebs-csi-controller
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      application: container-storage-interface
+      component: ebs-controller
+  template:
+    metadata:
+      {{ if eq .ConfigItems.kube_aws_iam_controller_kube_system_enable "false"}}
+      annotations:
+        iam.amazonaws.com/role: {{ .LocalID}}-ebs-csi-controller
+      {{ end }}
+      labels:
+        application: container-storage-interface
+        component: ebs-controller
+    spec:
+      containers:
+        - args:
+            - controller
+            - --endpoint=$(CSI_ENDPOINT)
+            - --logtostderr
+            - --v=5
+          env:
+            - name: CSI_ENDPOINT
+              value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
+          {{ if eq .ConfigItems.kube_aws_iam_controller_kube_system_enable "true"}}
+          # must be set for the AWS SDK/AWS CLI to find the credentials file.
+            - name: AWS_SHARED_CREDENTIALS_FILE # used by golang SDK
+              value: /meta/aws-iam/credentials.process
+          {{ end }}
+          image: registry.opensource.zalan.do/teapot/ebs-csi-driver:v0.5.0-master-2
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /healthz
+              port: healthz
+            initialDelaySeconds: 10
+            periodSeconds: 10
+            timeoutSeconds: 3
+          name: ebs-plugin
+          ports:
+            - containerPort: 9808
+              name: healthz
+              protocol: TCP
+          resources:
+            requests:
+              cpu: 50m
+              memory: 50Mi
+            limits:
+              cpu: 50m
+              memory: 50Mi
+          volumeMounts:
+            - mountPath: /var/lib/csi/sockets/pluginproxy/
```

Comment:
Same here, `.sock` files are usually in `/var/run`.

Reply:
This socket is not mounted from the host but is an `emptyDir` so node restarts should not be an issue.